### PR TITLE
:book: add info to tutorial about apis with examples

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**

Address the answer to common questions raised by giving a better explanation over APIs with examples.

### CODE REVIEW

**Review by darkowlzz — COMMENTED** [review]
I've suggested a few small improvements that I observed. Hope they are useful.
Thanks for more explanations in the docs. 🙂

File: docs/book/src/cronjob-tutorial/gvks.md [hunk 1]
```diff
@@ -46,6 +46,22 @@ a package.
 Now that we have our terminology straight, we can *actually* create our
 API!
 
+## So, how can we create our API? 
+
+In the next section, [Adding a new API](./cronjob-tutorial/new-api.html) we will check how the tool help us to create our own API's with the command `kubebuilder create api`.
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply docs/book/src/cronjob-tutorial/gvks.md:51]: done

File: docs/book/src/cronjob-tutorial/gvks.md [hunk 2]
```diff
@@ -46,6 +46,22 @@ a package.
 Now that we have our terminology straight, we can *actually* create our
 API!
 
+## So, how can we create our API? 
+
+In the next section, [Adding a new API](./cronjob-tutorial/new-api.html) we will check how the tool help us to create our own API's with the command `kubebuilder create api`.  
+
+The goal of this command is to create Custom Resource (CR) and Custom Resource Definition (CRD) for the our Kind(s). To check it further see; [Extend the Kubernetes API with CustomResourceDefinitions][[kubernetes-extend-api]].
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply docs/book/src/cronjob-tutorial/gvks.md:53]: done

File: docs/book/src/cronjob-tutorial/gvks.md [hunk 3]
```diff
@@ -46,6 +46,22 @@ a package.
...
 
+## So, how can we create our API? 
+
+In the next section, [Adding a new API](./cronjob-tutorial/new-api.html) we will check how the tool help us to create our own API's with the command `kubebuilder create api`.
+
+The goal of this command is to create Custom Resource (CR) and Custom Resource Definition (CRD) for our Kind(s). To check it further see; [Extend the Kubernetes API with CustomResourceDefinitions][kubernetes-extend-api].
+
+## But, why create APIs at all?
+
+Our APIs and resouces represent our solutions on the clusters. Basically, the CRDs are a definition of our customized Objects, and the CRs are an instance of it.
```
Comment (coderanger) [inline docs/book/src/cronjob-tutorial/gvks.md:57]: I think this section could use a bit more words.

```
New APIs are how we teach Kubernetes about our custom objects. The Go structs are used to generate a Custom Resource Definition (CRD) which includes the schema for our data as well as tracking data like what our new type is called. We can then create instances of our custom objects to use with (controllers)[some link todo].
```

Comment (camilamacedo86) [reply docs/book/src/cronjob-tutorial/gvks.md:59]: done

**Pull request merged**